test/OracleManagerApp.test.js

```diff
@@ -47,64 +47,69 @@ contract('OracleManagerApp', (accounts) => {
     const MANAGE_DATA_FEEDS = await oracleManagerApp.MANAGE_DATA_FEEDS()
     await acl.createPermission(root, oracleManagerApp.address, MANAGE_DATA_FEEDS, root);
-    //
+    // //
     await dataFeed1.initialize(root)
     await dataFeed2.initialize(root)
     await medianDataFeed.initialize([dataFeed1.address, dataFeed2.address], oracleManagerApp.address)
   })
 
   describe('initialize()', () => {
     it('initializes the contract', async () => {
-      await oracleManagerApp.initialize([dataFeed1.address, dataFeed2.address], medianDataFeed.address)
+      await oracleManagerApp.initialize(medianDataFeed.address)
       expect(await oracleManagerApp.hasInitialized()).to.equal(true)
     })
 
     it('reverts if called a second time', async () => {
-      await oracleManagerApp.initialize([dataFeed1.address, dataFeed2.address], medianDataFeed.address)
+      await oracleManagerApp.initialize(medianDataFeed.address)
       return assertRevert(async () => {
-        await oracleManagerApp.initialize([dataFeed1.address, dataFeed2.address], medianDataFeed.address)
+        await oracleManagerApp.initialize(medianDataFeed.address)
       })
     })
 
     it('sets the correct medianDataFeed', async () => {
-      await oracleManagerApp.initialize([dataFeed1.address, dataFeed2.address], medianDataFeed.address)
+      await oracleManagerApp.initialize(medianDataFeed.address)
       expect(await oracleManagerApp.medianDataFeed()).to.equal(medianDataFeed.address)
     })
 
-    it('sets the correct approvedDataFeedsLength', async () => {
-      await oracleManagerApp.initialize([dataFeed1.address, dataFeed2.address], medianDataFeed.address)
-      expect((await oracleManagerApp.approvedDataFeedsLength()).toNumber()).to.equal(2)
-    })
-
-    it('sets all dataFeeds to true in approvedDataFeeds mapping', async () => {
-      await oracleManagerApp.initialize([dataFeed1.address, dataFeed2.address], medianDataFeed.address)
-      expect(await oracleManagerApp.approvedDataFeeds(dataFeed1.address)).to.equal(true)
-      expect(await oracleManagerApp.approvedDataFeeds(dataFeed2.address)).to.equal(true)
-    })
-
-    it('emits DataFeedAdded events for initial dataFeeds', async () => {
-      const { logs } = await oracleManagerApp.initialize([dataFeed1.address, dataFeed2.address], medianDataFeed.address)
-      expect(logs[0].event).to.equal('DataFeedAdded')
-      expect(logs[1].event).to.equal('DataFeedAdded')
-      expect(logs[0].args.dataFeedAddress).to.equal(dataFeed1.address)
-      expect(logs[1].args.dataFeedAddress).to.equal(dataFeed2.address)
-    })
-
-    it('reverts if duplicate dataFeeds are submitted', async () => {
-      return assertRevert(async () => {
-        await oracleManagerApp.initialize([dataFeed1.address, dataFeed2.address, dataFeed1.address], medianDataFeed.address)
-      })
-    })
+    // Renable adding data feeds on initialize when
```

Comment:
would be awesome to have a `TODO:` here. convenient in later stages to see how many `TODO`'s we have left